composer.json

```diff
@@ -17,14 +17,20 @@
       "email": "patrick@maatwebsite.nl"
     }
   ],
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/77web/laravel-doctrine-orm.git"
+    }
+  ],
   "require": {
     "php": "^7.4|^8",
     "doctrine/migrations": "^3.4",
     "doctrine/dbal": "^2.10.1|^3",
-    "illuminate/config": "^6.0|^7.0|^8.0|^9.0",
-    "illuminate/contracts": "^6.0|^7.0|^8.0|^9.0",
-    "illuminate/console": "^6.0|^7.0|^8.0|^9.0",
-    "laravel-doctrine/orm": "^1 | ^2@dev"
```

Comment:
You can revert this change now (and remove the `repositories` part) :)

Reply:
Same with the PR in extensions repo

Reply:
done 😄

Reply:
Should still allow 1^, so `^1 | ^2@dev`  is correct here :)